Dockerfile

```diff
@@ -25,7 +25,24 @@ RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources
 RUN apt-get update
 RUN apt-get install -y --no-install-recommends mariadb-server mariadb-client
 RUN apt-get install -y --no-install-recommends nginx nginx-extras zip unzip cron nodejs npm yarn mc htop openssh-server git
-RUN apt-get install -y --no-install-recommends "php${PHP_VER}"-{fpm,mysql,curl,intl,gd,bcmath,soap,ldap,imap,xml,cli,zip,json,opcache,mbstring,apcu,imagick}
+# RUN apt-get install -y --no-install-recommends "php${PHP_VER}"-{fpm,mysql,curl,intl,gd,bcmath,soap,ldap,imap,xml,cli,zip,json,opcache,mbstring,apcu,imagick}
+RUN apt-get install -y --no-install-recommends "php${PHP_VER}"-fpm
```

Comment:
what does it change?

Reply:
apt doesn't picking regexp for some stranger things..and just edited to install one to one module. I could also install all with a single call to apt like: apt install php-fpm php-mysql etc..

Reply:
this is a better option
 apt install php-fpm php-mysql etc..

Reply:
sure. I will correct and commit this in some hours